rasa/core/domain.py

```diff
@@ -656,10 +656,10 @@ def cleaned_domain(self) -> Dict[Text, Any]:
         domain_data = self.as_dict()
         for idx, intent_info in enumerate(domain_data["intents"]):
             for name, intent in intent_info.items():
-                if intent.get("use_entities"):
+                if intent.get("use_entities") is True:
                     intent.pop("use_entities")
                 if not intent.get("ignore_entities"):
-                    intent.pop("ignore_entities")
+                    intent.pop("ignore_entities", None)
```

Comment:
Why did you add the `None` as default return value, if the return value is ignored?

Reply:
cause `pop` throws an error in case the key is not there